Makefile

```diff
@@ -0,0 +1,12 @@
+build: 
+	chmod +x pre-run.sh
+	./pre-run.sh first_deploy
+
+start:
+	chmod +x pre-run.sh
+	./pre-run.sh
+
+clean:
+	echo Deletes all local Kubernetes cluster. This command deletes the VM, and removes all associated files.
+	minikube stop
+	minikube delete --all
```

Comment:
How about creating a minikube profile which allows us to manage multiple minikube instances. 
That way we don't terminate other minikube instances (applications running) by executing `minikube stop`. 
```shell
minikube start -p docker-infrastructure
minikube profile docker-infrastructure
```
And perhaps add a `stop` step, that simply stops the current instance and doesn't delete it.